test/index.js

```diff
@@ -484,12 +510,70 @@ describe('render()', () => {
         });
     });
 
-    it('returns a promise when no options or callback given (server)', () => {
+    it('does not pass the request to the global context function (server)', (done) => {
```

Comment:
Was there a reason for this test? There's an [issue](#115) asking to have the `request` object available to the global context